parlai/tasks/natural_questions/text_utils.py

```diff
@@ -0,0 +1,144 @@
+# coding=utf-8
```

Comment:
@stephenroller looks like this apache license is super hands-off, are we good with direct copy/paste from another repo?

@mojtaba-komeili was there a `NOTICE` file where this was copied from? according to the license you'll need to include that here if so...

Reply:
I don't see any notice coming with it. Here is the link to the original repo: https://github.com/google-research-datasets/natural-questions

Reply:
ahh ok, I do see a LICENSE file that we might need to include

Reply:
@stephenroller I also get the following three lint errors related to licensing issue from this file:
```
[PAI201] Missing copyright `Copyright (c) Facebook, Inc. and its affiliates.`
[PAI202] Missing copyright `This source code is licensed under the MIT license found in the`
[PAI203] Missing copyright `LICENSE file in the root directory of this source tree.`
```

Reply:
Do we copy paste their code? We only need to bring in their license if we adapt their code.

If that's so then put our copyright at the top and then in the section adapted from them, delineate it in comments and paste their license

Reply:
The whole code is copied from them. From what you say, I put our copyright on top, and then at some point, I draw a line commenting that anything below here is copied from <link>.
I also include their license file with it (both in a subdirectory in the task dir).

Reply:
Yeah us at top, a big line, and then them.